Functions/BIDS/xASL_bids_MergeNifti.m

```diff
@@ -41,7 +41,7 @@
 %
 % -----------------------------------------------------------------------------------------------------------------------------------------------------
 % __________________________________
-% Copyright 2015-2021 ExploreASL
+% Copyright 2015-2022 ExploreASL
```

Comment:
Very nice work! Is it easy for you to add a list in the function header of example cases? It's GE and Siemens right?

Reply:
Not sure if I follow - the header is already quite detailed. Explaining what is done for GE and Siemens. What exactly you want to add?

Reply:
OK I see now. I thought a bit more detail, e.g.:

* dcm2nii outputs GEs deltaM and M0scan separately, even though they are scanned in a single sequence and should belong together
* Siemens stores all controls and labels separately, which need to be reordered according their acquisition order

BTW: I would move the sub-functions to actual subfunctions, instead of separate code. If a function is only used in a single function, and belongs with it, then I would use it as true subfunction (as is the case here). The only disadvantage may be debugging, but the code tree could become definitely easier to follow! 

(fine if you resolve this and do this in a new issue, though adding my text above, adding some more of your own, and moving to subfunctions is perhaps not a lot of work)

Reply:
Detail added.
Subfunctions merged.